components/director/internal/oathkeeper/model_test.go

```diff
@@ -42,232 +37,6 @@ func TestAuthFlow(t *testing.T) {
 	})
 }
 
-func TestReqData_GetAuthID(t *testing.T) {
```

Comment:
Deleting might be reasonable for some of these tests but also some flows are probably no longer covered. The test coverage for the _model.go_ file from this PR is about 60%. For comparison, from the main brach it is 85%. Consider adding some tests.

Reply:
GetAuthId was split into the Match methods of ObjectContextProviders. All test cases were moved to the coresponding test files.  I will add some tests for ExtractCoordinates.